package.json

```diff
@@ -6,6 +6,7 @@
     "classnames": "^2.3.2",
     "date-fns": "^2.29.3",
     "dompurify": "^3.0.2",
+    "moment": "^2.29.4",
```

Comment:
Why do we need both **date-fns** and **moment**? Please use only one. As moment is deprecated, my suggestion would be to go with date-fns